### PR TITLE
Fix some issues with Kamaji extension

### PR DESCRIPTION
**Pull request by aalves08**
This PR fixes some issues I've detected while doing tests with Rancher Extensions for the upcoming Rancher 2.9 release:

- a routing issue (hard refresh on product route wouldn’t work - fixes to `product.ts` and `routes.ts`)
- layout issue (sidebar disappears - removal of `layout: "single"` in `Dashboard` component)
- product name needs to be a single word like `kamaji`  not `multitenancy management`
- updated shell package dependency to latest stable version
- fixes some linting issues
- version bumping to prepare for a release

To test:
- check out branch and yarn install dependencies
- do `yarn build-pkg kamaji` + `yarn serve-pkgs` to serve the locally built assets and then go to your Rancher instance and do a developer load
https://extensions.rancher.io/extensions/extensions-getting-started#test-built-extension-by-doing-a-developer-load

**Before**
before

**After**
after

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "rancher-clastix",
-  "version": "0.1.0",
+  "version": "0.1.3",
```
Comment (prometherion) [inline package.json]: I would bump to `0.2.0` honestly.

File: pkg/kamaji/types.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-export const CLASTIX_PRODUCT_NAME = "Multitenancy Management";
-export const CLASTIX_NAME = "kamaji";
+export const CLASTIX_DASHBOARD = 'dashboard';
+export const CLASTIX_PRODUCT_NAME = 'kamaji';
```
Comment (prometherion) [inline pkg/kamaji/types.ts:2]: @bsctl could have some remarks on this renaming, he'll be back next week.

**Review by prometherion — CHANGES_REQUESTED** [review]

File: pkg/kamaji/types.ts (hunk 2 above)
Comment (bsctl) [reply pkg/kamaji/types.ts:2]: +1

**Comment by aalves08** [comment]
@bsctl are there any extra changes besides bumping the version to `0.2.0` that you would like me to do?

**Comment by bsctl** [comment]
@aalves08 
> @bsctl are there any extra changes besides bumping the version to `0.2.0` that you would like me to do?

It should be fine, please bump to `0.2.0`

**Comment by aalves08** [comment]
@bsctl @prometherion change done. Requesting final approval 🙏

**Review by bsctl — APPROVED** [review]
lgtm

**Pull request merged**